PATTERNS.md

```diff
@@ -107,16 +114,16 @@ The Pattern Type of an Anything-But Pattern is
 of strings.  It will match a string value which
 is not equal to any of the strings in the array.
 
-If a Field in a Pattern contains an Anything-But Pattern, 
+If a Field in a Pattern contains an Anything-But Pattern,
```

Comment:
nit: `Anything-But`

Reply:
I think that's the way we're spelling it.  I'm missing something?

Reply:
Ah, got it!